bokeh/model/data_model.py

```diff
@@ -44,6 +44,10 @@
 class DataModel(Model):
     __data_model__ = True
 
+    # explicit __init__ to support Init signatures
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
```

Comment:
This is unfortunate, but if there is no better way, then we will have live with this. On the other hand, something similar (though for static typing) had to be done in bokehjs, due to TypeScript's limitations (though it may be possible to generate those constructors with AST transforms).

Reply:
Generating a (closure) `__init__` and attaching it does not allow `super` to function correctly, which is not really surprising. I did briefly toy with iterating over the MRO manually but that seems like a solution worse than the problem.